README.md

```diff
@@ -26,6 +26,7 @@ For many people Docker is not an option, but Singularity is. Most Docker contain
 | [BBTools](https://hub.docker.com/r/staphb/bbtools/) <br/> [![docker pulls](https://img.shields.io/docker/pulls/staphb/bbtools.svg?style=popout)](https://hub.docker.com/r/staphb/bbtools) | <ul><li>38.76</li><li>38.86</li></ul> | https://jgi.doe.gov/data-and-tools/bbtools/ |
 | [bcftools](https://hub.docker.com/r/staphb/bcftools/) <br/> [![docker pulls](https://img.shields.io/docker/pulls/staphb/bcftools.svg?style=popout)](https://hub.docker.com/r/staphb/bcftools) | <ul><li>1.10.2</li><li>1.11</li><li>1.12</li></ul> | https://github.com/samtools/bcftools |
 | [bedtools](https://hub.docker.com/r/staphb/bedtools/) <br/> [![docker pulls](https://img.shields.io/docker/pulls/staphb/bedtools.svg?style=popout)](https://hub.docker.com/r/staphb/bedtools) | <ul><li>2.29.2</li><li>2.30.0</li></ul> | https://bedtools.readthedocs.io/en/latest/ <br/>https://github.com/arq5x/bedtools2 |
+| [bowtie2](https://hub.docker.com/r/staphb/bowtie2/) <br/> [![docker pulls](https://img.shields.io/docker/pulls/staphb/bowtie2.svg?style=popout)](https://hub.docker.com/r/staphb/bowtie2) | <ul><li>2.29.2</li><li>2.30.0</li></ul> | http://bowtie-bio.sourceforge.net/bowtie2/manual.shtml <br/>https://github.com/BenLangmead/bowtie2 |
```

Comment:
The middle column of the table should be: `<ul><li>2.4.4</li></ul>`  To match the version of bowtie2 included in the docker image